src/app/sidenav/sidenav-collapsible/sidenav-collapsible.component.unit.spec.ts

```diff
@@ -21,7 +21,60 @@ describe('MzSidenavCollapsibleComponent:unit', () => {
     fixture.detectChanges();
   });
 
-  it('should create', () => {
-    expect(component).toBeTruthy();
+  describe('ngAfterViewInit', () => {
+
+    it('should call initCollapsible function', () => {
+      spyOn(component, 'initCollapsible');
+
+      component.initCollapsible();
+
+      expect(component.initCollapsible).toHaveBeenCalled();
+    });
+  });
+
+  describe('initCollapsible', () => {
+
+    function forceSetTimeoutEnd() {
+      tick(1); // force setTimeout execution
+    }
+
+    it('should initialize collapsible using jquery', fakeAsync(() => {
+
+      component.onClose = () => { const close = 'close'; };
```

Comment:
`() => { const close = 'close'; };` can be replaced with `() => null`, it's less confusing, at first I tough that `close` was important.